worker/pkg/workflows/datasync/activities/gen-benthos-configs/processors.go

```diff
@@ -366,21 +366,12 @@ func constructBenthosJsProcessor(jsFunctions, benthosOutputs []string) string {
 
 	jsCode := fmt.Sprintf(`
 (() => {
-function setNestedProperty(obj, path, value) {
-	path.split('.').reduce((current, part, index, parts) => {
-		if (index === parts.length - 1) {
-			current[part] = value;
-		} else {
-			current[part] = current[part] ?? {};
-		}
-		return current[part];
-	}, obj);
-}
 %s
 const input = benthos.v0_msg_as_structured();
 const output = { ...input };
```

Comment:
Is the output variable used anymore then?
Does it make sense to rename `updatedValues` to `output`? `patchedOutput`? `updatedValues`? `patchedValues`?